draft-ietf-quic-multipath.md

```diff
@@ -252,41 +252,39 @@ the use of the multipath extension during the connection handshake,
 as specified in {{QUIC-TRANSPORT}}. The new transport parameter is
 defined as follows:
 
-- initial_max_paths (current version uses 0x0f739bbc1b666d07): the
-  initial_max_paths transport parameter is included if the endpoint supports
+- initial_max_path_id (current version uses 0x0f739bbc1b666d09): the
+  initial_max_path_id transport parameter is included if the endpoint supports
   the multipath extension as defined in this document. This is
-  a variable-length integer value specifying the maximum number of
-  active concurrent paths an endpoint is willing to maintain.
-  The value of the initial_max_paths parameter MUST be at least 2.
+  a variable-length integer specifying the initial maximum value of
+  available path identifier an endpoint is willing to support.
+  The value of the initial_max_path_id parameter MUST be at least 2.
   An endpoint that receives a value less than 2 MUST close
   the connection with an error of type TRANSPORT_PARAMETER_ERROR.
```

Comment:
However, I'm not sure if we really need to close the connection if initial_max_path_id=0. I think this would simply mean that you don't allow for any additional path yet.

Reply:
If endpoints don't want to allow additional paths, endpoints could choose not to send the transport parameter. 
There's only one extra scenario: Endpoint doesn't want to allow any additional paths when handshake is finished, but keep the possibility to allow for new paths with MAX_PATH_ID frame when necessary. That could be a possible strategy.

Reply:
I record this issue in #381 . If people want to add this mechanism, we could add it in a separate PR.